plugins/DataLoadRlog/rlog_parser.cpp

```diff
@@ -69,13 +69,26 @@ bool RlogMessageParser::parseMessageImpl(const std::string& topic_name, capnp::D
     case capnp::DynamicValue::STRUCT: 
     {
       auto structValue = value.as<capnp::DynamicStruct>();
-      for (auto field : structValue.getSchema().getFields()) 
+      std::string struct_name;
+      KJ_IF_MAYBE(e_, structValue.which()) { struct_name = e_->getProto().getName(); }
+      for (auto field : event.getSchema().getNonUnionFields())
```

Comment:
```suggestion
      for (auto &field : event.getSchema().getNonUnionFields())
```

Reply:
Did for both since the other wasn't using a const